server/routes/overview/overviewController.ts

```diff
@@ -18,9 +18,10 @@ export default class OverviewController {
   }
 
   getSupportNeedsView: RequestHandler = async (req, res, next): Promise<void> => {
+    const { prisonNumber } = req.params
     const { prisonerSummary } = req.session
-    const { supportNeeds } = req.session
 
+    const supportNeeds = await this.curiousService.getPrisonerSupportNeeds(prisonNumber, req.user.username)
```

Comment:
👍 
The more I think about it, the more I think this the right way to do data like this (for now at least)
By not storing it in the session and retrieving it every time, it means that if curious is temporarily down, then trying again shortly afterwards would work (if curious was back up), and would not get the duff curious data from the session.

Efficient to call it on each page load? Perhaps not, though that is unproven at this stage. We can consider some kind of caching as and when this is proven to be a problem 👍

Reply:
Yeah I think we might want to look at some caching at some stage.